### ISSUE TITLE
Not Loading stopped playing files

### ISSUE THREAD

**Issue by marioandluigi878** [Kalejin/DCSB#13]
Just stopped loading the program no error no nothing and for the sound files I got red dots and now just it won't load up at all

**Comment by Kalejin** [comment]
Can you please mouse over those red dots? It should show a pop up with error message.

**Comment by marioandluigi878** [comment]
I wish i could but the program isn't even loading up for me anymore which is such a shame because I really like this program too

**Comment by marioandluigi878** [comment]
[config - Copy.txt](https://github.com/Kalejin/DCSB/files/1723739/config.-.Copy.txt)
Github doesn't like XML so I just quickly made it .txt

**Comment by Kalejin** [comment]
Seems like the problem is in output device. Output device that has been set is not available anymore and application does not handle it.

I'll fix it, but for now you can try replacing 2 to 0 in following section of the config and run the app again.
```xml
...
<PrimaryOutputDevice>
    <Number>2</Number>
</PrimaryOutputDevice>
...
```

**Comment by marioandluigi878** [comment]
That seemed to work.....

**Comment by Kalejin** [comment]
Handling connecting and disconnecting output devices is buggy, so it'll happen again.
I'll reopen this to track me fixing this issue.

**Comment by Kalejin** [comment]
Resolved in commit #0c9549e
